src/main/java/com/dev/cinema/validation/FieldsValueMatchValidator.java

```diff
@@ -0,0 +1,22 @@
+package com.dev.cinema.validation;
+
+import java.util.Objects;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class FieldsValueMatchValidator implements ConstraintValidator<FieldsValueMatch, Object> {
```

Comment:
Instead of object - you can use UserRequestDto.
Accordingly - change line 17

Reply:
As I mentioned earlier, this is just more general implementation, so I'm using Object so that validation could be applied not just for user and not just for password but for other fields and models as well.